<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-F5U5V1J" failures="1" tests="3" name="Test" time="113.413" errors="0" timestamp="2022-09-03T17:50:40 IST">
  <testcase classname="com.Neostox.TestCases.LoginPageTest" name="LoginPageNavigationTest" time="23.418"/>
  <testcase classname="com.Neostox.TestCases.LoginPageTest" name="Logout" time="0.005"/>
  <testcase classname="com.Neostox.TestCases.LoginPageTest" name="testcas1" time="0.012">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.Neostox.TestCases.LoginPageTest.testcas1(LoginPageTest.java:37)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- testcas1 -->
</testsuite> <!-- Test -->
